Update README (#660)
BlueBrain/nexus-web: README.md

  </a>
</p>

> <p align="center">Nexus Web (soon to be renamed as Nexus Fusion) is the interface of Blue Brain Nexus, the open-source knowledge graph for data-driven science.</p>

Reviewer: ```suggestion
<p align="center">Nexus Fusion (previously Nexus Web) is the interface of Blue Brain Nexus, the open-source knowledge graph for data-driven science.</p>
```

Let's talk about the change happening in the past since the release already happened.
